server/src/index.php

```diff
@@ -3,7 +3,7 @@
 use \Psr\Http\Message\ServerRequestInterface as Request;
```

Comment:
I think the front-controller should be in a `public` directory and not in `src`. `public` can either be inside `src` or at the same level.

Reply:
agreed.